app/jobs/broadcast_quiz_job.rb

```diff
@@ -1,7 +1,13 @@
 class BroadcastQuizJob < ApplicationJob
   queue_as :default
 
-  def perform(message)
-    ActionCable.server.broadcast 'quiz_channel', message
+  def perform(data)
+    ActionCable.server.broadcast 'quiz_channel', create_partial(data)
+  end
+
+  def create_partial(data)
+    # @answer = Answer.new(quiz_id: data.quiz_id, question_id: data.question_id)
+    @quiz = Quiz.find(data[:quiz_id])
```

Comment:
yeah that needs to be removed on cleanup. I was hoping those would end up in the partial, but no dice.